ophyd/sim.py

```diff
@@ -336,6 +336,8 @@ class SynGauss(SynSignal):
     noise_multiplier : float
         Only relevant for 'uniform' noise. Multiply the random amount of
         noise by 'noise_multiplier'
+    random_state : numpy random state object
```

Comment:
Can you add `, optional` after the type here? While you are at it, the parameters `noise` and `noise_multiplier` should be marked optional as well.

Reply:
good catch. I will add optional after the type.